packages/react/src/ReferenceInput/ReferenceInput.stories.tsx

```diff
@@ -18,3 +22,31 @@ export const FreeText = (): JSX.Element => (
     <ReferenceInput name="foo" />
   </Document>
 );
+
+const FishPatientProfileSD = FishPatientResources.getFishPatientProfileSD();
+
+export const PatientProfileAndPatient = (): JSX.Element => {
+  const medplum = useMedplum();
+  const [loaded, setLoaded] = useState(false);
+  useEffect(() => {
+    (async (): Promise<boolean> => {
```

Comment:
Will this create these resources every time the story is loaded?

Reply:
It really seems like it should be, but after adding some debug lines, no, it appears the resources available are cleared out when moving between stories. Now I'm curious too. @codyebberson could you point us at where the clearing of created resources happens when moving around in storybook?

Reply:
This is a good question.  I *think* (we should confirm) that the Storybook `preview.tsx` is re-evaluated per preview, which would mean that each preview has an independent `MockClient`

See: `packages/react/.storybook/preview.tsx`